src/nexmo/__init__.py

```diff
@@ -497,16 +510,22 @@ def get(self, host, request_uri, params=None, header_auth=False):
         logger.debug("GET to %r with params %r, headers %r", uri, params, headers)
         return self.parse(host, self.session.get(uri, params=params, headers=headers))
 
-    def post(self, host, request_uri, params, header_auth=False):
+    def post(self, host, request_uri, params, signature_auth=False, header_auth=False):
```

Comment:
not crazy about the name of the parameter here - it implies that your doing signature auth every time - maybe something like `supports_signature_auth` could make more sense in context - 'hey this API supports signature auth - but  we need to interrogate the request first'

Reply:
done